ccminer.vcxproj

```diff
@@ -434,6 +434,9 @@
     <CudaCompile Include="sha256\sha256d.cu" />
     <CudaCompile Include="sha256\cuda_sha256t.cu" />
     <CudaCompile Include="sha256\sha256t.cu" />
+    <CudaCompile Include="x12\x12.cu">
+      <CodeGeneration Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">compute_50,sm_50;compute_52,sm_52;compute_30,sm_30</CodeGeneration>

```

Comment:
this is useless, no cuda code in this file

Reply:
But all other cu without CUDA code compiles by CUDA compiler in vc project

Reply:
Oh, you mean "CudaGeneration" property?